substra/sdk/backends/local/compute/worker.py

```diff
@@ -307,6 +318,16 @@ def schedule_testtuple(self, tuple_):
                     container_volume=_VOLUME_MODELS_RO,
                 )
 
+            if self._db.is_local(dataset.key):
+                data_sample_paths = self._get_data_sample_paths_arg(
+                    _VOLUME_INPUT_DATASAMPLES['bind'],
+                    tuple_.dataset
+                )
+                command += f" --data-sample-paths {data_sample_paths}"
+            else:
+                command += " --fake-data"
+                command += f" --n-fake-samples {len(tuple_.dataset.keys)}"
```

Comment:
Shouldn't that be `tuple_.dataset.data_sample_keys` ?

Reply:
Good catch!

Reply:
Fixed!